jira_docugen.tex

```diff
@@ -74,55 +62,157 @@ \subsubsection{Test Items}\label{test-items}
   Data Facility-managed systems.
 \end{itemize}
 
-\subsubsection{Requirements}\label{requirements}
 
-\begin{itemize}
-\tightlist
-\item
-  \href{https://jira.lsstcorp.org/browse/LVV-139}{LVV-139} -
-  DMS-REQ-0308-V-01: Software Architecture to Enable Community Re-Use
-\end{itemize}
 
-\subsubsection{Precondition}\label{precondition}
+\subsubsection{Intercase Dependencies}
+None.
+
+
+\subsubsection{Environment Needs}
 
-- Input specification\\[2\baselineskip]No input data is required for
-this test case.
+\paragraph{Software}
+Software All prerequisite packages listed at
+https://pipelines.lsst.io/install/ prereqs/centos.html must be available
```

Comment:
Erroneous space between `install/` and `prereqs`.

Reply:
Corrected in jira.